### ISSUE TITLE
`mod` statements are ordered ASCIIbetically

### ISSUE THREAD

**Issue by xfix** [rust-lang/rustfmt#3242]
https://gitlab.com/KonradBorowski/advent-of-code-2018/commit/cd9f8773de229b8fd6356e6c722d0ef55173f809

```diff
diff --git a/src/main.rs b/src/main.rs
index 090e2e1..f147489 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -3,6 +3,7 @@ use std::io::{self, Read, Write};
 use structopt::StructOpt;

 mod day1;
+mod day10;
 mod day2;
 mod day3;
 mod day4;
@@ -11,7 +12,6 @@ mod day6;
 mod day7;
 mod day8;
 mod day9;
-mod day10;
 #[cfg(test)]
 mod testmacros;
```

I think it would be better to use natural ordering.

**Comment by topecongiro** [comment]
IMHO this hardly ever occurs in practice but the implementation cost is pretty high, so I am going to mark this as won't fix. Sorry.

**Comment by xfix** [comment]
Fixed by #3764.
